apps/frontend/src/libs/components/popover/popover.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+	useCallback,
+	useHandleClickOutside,
+	useRef,
+	useState,
+} from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+const ESCAPE_KEY = "Escape";
+
+type Properties = {
+	children: React.ReactNode;
+	content: React.ReactNode;
+};
+
+const Popover = ({ children, content }: Properties): JSX.Element => {
+	const [isOpen, setIsOpen] = useState<boolean>(false);
```

Comment:
```suggestion
	const [isOpened, setIsOpened] = useState<boolean>(false);
```

Reply:
I think both variants are ok, even `isOpen` might be better because "open" there is an adjective describing a state. "Opened" is more like a passive construction, and it describes the action/process of opening up. (ex. you say 'the shop is open right now', not 'the shop is opened right now'). But I see that in modal it was named like `isOpened`, so for consistency I'll do it.